### DISCUSSION TITLE
How well is support for Bleak & Kivy

### DISCUSSION THREAD

**Discussion by EdwinFairchild** [hbldh/bleak#1450] [Q&A]
I am wanting to start making a BLE app, I am leaning towards Kivy only because I would be able to use Bleak which I am familiar with. Question is how stable is it on Android/iOs ?  or if anyone has any insights on performance or other issues observed?

**Comment by dlech** [comment]
It has been a bit neglected but there has been some recent updates to get it working again. #883 really needs to be fixed, but other than that, I think it is working OK.
